specification/attributes/unit_format.md

```diff
@@ -0,0 +1,92 @@
+  * `<plural-units>` - "GB", "Seconds"
+  * `<singular-unit>-<plural-time-units>` - "GB-Hours", "MB-Days"
+  * `<plural-units>/<singular-time-unit>` - "GB/Hour", "PB/Day"
+* Units MAY be expressed with a unit quantity or time interval.  If a unit quantity or time interval is used, the unit quantity or time interval MUST be expressed as a whole number.  The following formats are valid:
+  * `<quantity> <plural-units>` - "1000 Tokens", "1000 Characters"
+  * `<plural-units>/<interval> <plural-time-units>` - "Units/3 Months"
+* Unit values and components of columns using the Unit Format MUST use a capitalization scheme that is consistent with the capitalization scheme used in this attribute, if that term is listed in this section. For example, a value of "gigabyte-seconds" would not be compliant with this specification as the terms "gigabyte" and "second" are listed in this section with the appropriate capitalization.  If the unit is not listed in the table, it is to be used over a functional equivalent with similar meaning with the same capitalization scheme.
+* Units SHOULD be composed of the list of recommended units listed in this section, unless the unit value covers a dimension not listed in the recommended unit set, or if the unit covers a count-based unit distinct from recommended values in the count dimension listed in this section.  
+
+### Data Size Unit Names
+
+Data size MUST be abbreviated using the following standard abbreviations.  Each data size abbreviation can be considered both the singular and plural form of the unit.  For example, "GB" is both the singular and plural form of the unit "GB".  The following table lists the valid data size units.
+
+| Data size in bits    | Data size in bytes    |
+| -------------------- | --------------------- |
+| Kib (kibibit = 2^10) | KiB (kibibyte = 2^10) |
+| Mib (mebibit = 2^20) | MiB (mebibyte = 2^20) |
+| Gib (gibibit = 2^30) | GiB (gibibyte = 2^30) |
+| Tib (tebibit = 2^40) | TiB (tebibyte = 2^40) |
+| Pib (pebibit = 2^50) | PiB (pebibyte = 2^50) |
+| Eib (exbibit = 2^60) | EiB (exbibyte = 2^60) |
+| Kb (kilobit = 10^3)  | KB (kilobyte = 10^3)  |
+| Mb (megabit = 10^6)  | MB (megabyte = 10^6)  |
+| Gb (gigabit = 10^9)  | GB (gigabyte = 10^9)  |
+| Tb (terabit = 10^12) | TB (terabyte = 10^12) |
+| Pb (petabit = 10^15) | PB (petabyte = 10^15) |
+| Eb (exabit = 10^15)  | EB (exabyte = 10^15)  |
+
+### Count-based Unit Names
+
+A count-based unit is a noun that represents a discrete number of items, events, or services.  For example, a count-based unit can be used to represent the number of requests, instances, tokens, or connections.  
+
+If the list of recommended values does not cover a count-based unit, a provider MAY introduce a new noun representing a count-based unit.  All nouns appearing in unit that are not listed in the recommended values table will be considered count-based units.  A new count-based unit value MUST be capitalized.
+
+| Count        |
+|--------------|
```

Comment:
Is this an exhaustive list, or is it merely a list of examples?  Either way, it would be good to specify.

Reply:
Resolving in favor of comment made on line 54.